packages/components/src/components/money/money.jsx

```diff
@@ -1,7 +1,8 @@
-import classNames    from 'classnames';
-import PropTypes     from 'prop-types';
-import React         from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
 import CurrencyUtils from '@deriv/shared/utils/currency';
```

Comment:
I know this is not part of the commit, but do you think it is possible to destructure this import for a better future? 
`import { getDecimalPart, isCryptocurrency } from '@deriv/shared/utils/currency';`

Reply:
Personally I prefer to use **utilities** under a wildcard imported object. It gives me the feel of using static classes of OOP languages :)
Sometimes we're looking for some functionality we think it's already implemented in utils but we don't know the exact name. In this case intellisense will help under a wildcard.
Both approaches have pros and cons.
Please let me know if there is any project preferences on this.

Reply:
The only advantage I can think of is tree-shaking.